model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceACL.scala

```diff
@@ -15,6 +15,11 @@ case class WorkspaceACLUpdateResponse(subjectId: String, accessLevel: WorkspaceA
 
 case class WorkspaceACLUpdateResponseList(usersUpdated: Seq[WorkspaceACLUpdateResponse], invitesSent: Seq[WorkspaceACLUpdate], invitesUpdated: Seq[WorkspaceACLUpdate], usersNotFound: Seq[WorkspaceACLUpdate])
 
+case class WorkspaceCatalog(email: String, catalog: Boolean)
+case class WorkspaceCatalogResponse(subjectId: String, catalog: Boolean)
+
+case class WorkspaceCatalogUpdateResponseList(usersUpdated: Seq[WorkspaceCatalogResponse], usersNotFound: Seq[String])
```

Comment:
I prefer a name like `emailsNotFound` to make it explicit that this is an email and not a Subject ID